doc/source/operations/secret-rotation.rst

```diff
@@ -234,6 +234,28 @@ Full method
 
           grafana-cli admin reset-admin-password --password-from-stdin
 
+    .. note::
```

Comment:
Could you please add this to the regular instructions rather than as a note? I think we should just get everyone to check the user ID, rather than letting some people get the error and have to go back and do it again

Reply:
Sounds good.